sources/packages/backend/apps/db-migrations/src/sql/ApplicationOfferingChangeRequests/Create-application-offering-change-requests.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE sims.application_offering_change_requests (
+  id SERIAL PRIMARY KEY,
+  application_id INT NOT NULL REFERENCES sims.applications(id),
+  change_offering_id INT NOT NULL REFERENCES sims.education_programs_offerings(id),
```

Comment:
I know i created the DB column names :) , but I feel it would be better to have change_offering_id as requested_offering_id? Thoughts? @andrewsignori-aot @dheepak-aot @andrepestana-aot @sh16011993